test/e2e/testutils.go

```diff
@@ -0,0 +1,132 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	operator "github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
+
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 30
+)
+
+// createCatalogSourceConfig returns a CatalogSourceConfig based on the provided parameters
+func createCatalogSourceConfig(catalogSourceConfigName string, namespace string, targetNamespace string, packages string) *operator.CatalogSourceConfig {
```

Comment:
Same comment as the one about createOperatorSource

Reply:
Please see comment on createOperatorSource.